app/src/main/java/de/grobox/transportr/networks/TransportNetworks.kt

```diff
@@ -511,6 +511,18 @@ private val networks = arrayOf(
                     )
                 )
             ),
+            Country(
+                R.string.np_name_pt, flag = "🇵🇹", networks = listOf(
```

Comment:
I think this should be `R.string.np_region_pt` as it is the country name and not the network's one.

Reply:
@ialokim Yes, I agree, done.